main.c

```diff
@@ -29,13 +29,17 @@
 
 extern app_subsys redsocks_subsys;
 extern app_subsys base_subsys;
+#ifndef ANDROID
```

Comment:
if redudp is removed just because lacking of support of tsearch/tfind, i do not think redudp should be disabled for android. Some people may need it.